addons/mail/security/mail_security.xml

```diff
@@ -27,6 +27,20 @@
             <field name="domain_force">[(1, '=', 1)]</field>
         </record>
 
+        <record id="discuss_gif_favorite_rule" model="ir.rule">
+            <field name="name">Discuss GIF favorite</field>
+            <field name="model_id" ref="model_discuss_gif_favorite"/>
+            <field name="groups" eval="[Command.link(ref('base.group_user'))]"/>
+            <field name="domain_force">[('create_uid', '=', user.id)]</field>
+        </record>
+
+        <record id="discuss_gif_favorite_admin" model="ir.rule">
```

Comment:
inconsistent `id` naming

- `rule` keyword or not inside name? (as xml id are common to all type of records, I'd suggest having `rule` inside the name can help)
- group name inside name? probably yes -> missing `user` in the first one, having `admin` in the other is fine